### ISSUE TITLE
Application is reported as broken in Yunohost 4.3.4.1

### ISSUE THREAD

**Issue by prog-amateur2** [YunoHost-Apps/jellyfin_ynh#83]
Hello on Yunohost 4.3.4.1, after each diagnostic, I can see the following message : 

```
A problem has been detected for the application Jellyfin

    This application is currently reported as broken in the YunoHost application catalog. This may be a temporary problem. Until the maintainers attempt to resolve the issue, updating of this application is disabled.
```

This message happens for quite a long time now, so I would like to know if you still maintain this app, and if possible, if you could fix this issue please ? 

Thank you very much

**Comment by tituspijean** [comment]
Yes, cf. currently pinned issues and opened PR.

**Comment by Salamandar** [comment]
Should be fixed by #87
